src/main/webapp/resources/js/components/alerts/WarningAlert.jsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Component to render a Warning alert with an icon
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Alert } from "antd";
+
+export function WarningAlert(props) {
```

Comment:
To all of these, can you use the following pattern:

```js
export function InfoAlert({ message, description, ...props }) {
  return (
    <Alert
      type="info"
      showIcon
      message={message}
      description={description}
      {...props}
    />
  );
}
```

Reply:
Sure!

Reply:
Done in 82cbf1d540c536326e9a74e802ccc1fd825abbf7